sql/src/SqlCsv.sk

```diff
@@ -807,6 +807,7 @@ mutable class DiffParsingState(
 )
 
 fun nack(writer: mutable Debug.BufferedWriter, diff: FinalTableDiff): void {
+  SKDBMonitor.addReplicationWrites(1 + diff.lines.size());
```

Comment:
It's a little confusing to call these writes - although I see why you did. These are 'rejected rows'. And I would rename reads to be 'processed rows' perhaps.